src/ort_genai_c.cpp

```diff
@@ -246,6 +276,63 @@ OgaResult* OGA_API_CALL OgaTokenizerStreamDecode(OgaTokenizerStream* p, int32_t
   OGA_CATCH
 }
 
+OgaResult* OGA_API_CALL OgaCreateStrings(OgaStrings** out) {
+  OGA_TRY
+  *out = reinterpret_cast<OgaStrings*>(std::make_unique<std::vector<std::string>>().release());
+  return nullptr;
+  OGA_CATCH
+}
+
+OgaResult* OGA_API_CALL OgaCreateAllocatedStrings(size_t num_strings, size_t* str_lengths, OgaStrings** out) {
```

Comment:
The problem with this API is that there is a chance of discrepancy betwen num_strings and the actual number of lenghts. With a one by one approach, there is not such a change.

Reply:
Removed OgaStrings type.